10.9-libcxx/stable/main/finkinfo/base/libncurses6-shlibs.info

```diff
@@ -0,0 +1,113 @@
+Package: libncurses6-shlibs
+Version: 6.0
+Revision: 1
+
+GCC: 4.0
+Source: ftp://ftp.gnu.org/pub/gnu/ncurses/ncurses-%v.tar.gz
+Source-MD5: ee13d052e1ead260d7c28071f46eefb1
+SourceDirectory: ncurses-%v
+BuildDepends: fink (>= 0.13.0)
+Depends: base-files
+Essential: true
+PatchScript: <<
+#!/bin/sh -ex
+	# apply upstream patchsets
+	for p in patches/*.patch.gz; do
+		gzip -cd $p | patch -p1
+	done
+
+	# Fix warnings in clang by removing some useless compiler
+	# switches (they are not useful for gcc either).
+	perl -pi -e "s/-no-cpp-precomp//" configure
+
+	#Need to remove these entries to avoid failure on case sensitive file systems
+	perl -pi -e "s/hp70092a\|//; s/hp2621a\|//; s/2621a\|//" misc/terminfo.src
+<<
+ConfigureParams: <<
+  --without-ada \
+  --without-pkg-config \
+  --with-shared \
+  --with-debug \
+  --enable-termcap \
+  --mandir=%p/share/man \
+  --with-terminfo-dirs="%p/share/terminfo:/usr/share/terminfo" \
+  --with-default-terminfo-dir="%p/share/terminfo" \
+  --enable-no-padding \
+  --enable-sigwinch \
+  --enable-tcap-names \
+  --enable-hard-tabs \
+  --enable-overwrite \
+  --disable-mixed-case \
+  --libdir=%p/lib/ncurses \
+  cf_cv_typeof_chtype=long \
+  cf_cv_gcc_inline=no
+<<
+NoSetCFLAGS: true
+NoSetLDFLAGS: true
+NoSetCPPFLAGS: true
+UseMaxBuildJobs: false
+InstallScript: <<
+#!/bin/sh -ev
+  make install DESTDIR=%d DYLD_LIBRARY_PATH=`pwd`/lib:`pwd`/lib/ncurses
+# add extra symlinks in the main lib directory
+  ln -s ncurses/libform.6.dylib %i/lib/libform.dylib
+  ln -s ncurses/libmenu.6.dylib %i/lib/libmenu.dylib
+  ln -s ncurses/libncurses.6.dylib %i/lib/libncurses.dylib
+  ln -s ncurses/libpanel.6.dylib %i/lib/libpanel.dylib
+  ln -s ncurses/libncurses.6.dylib %i/lib/libcurses.dylib
+  ln -s ncurses/libncurses.6.dylib %i/lib/libtermcap.dylib
+<<
+SplitOff: <<
+ Package: libncurses6
+ Depends: libncurses6-shlibs (= %v-%r)
+ BuildDependsOnly: true
+ Conflicts: ncurses-dev, libncurses5-64bit, libncurses5
+ Replaces: ncurses-dev, ncurses (<= 5.3-20031018-2), libncurses5-64bit, ncurses (<= 5.7-20100227-1), libncurses5
+ Provides: libncurses6-dev
```

Comment:
There was a libncurses5:Provides:libncurses5-dev since long ago; do we really need this cargo-culting if we are starting with a new libversion? "foo/foo-shlibs" is a well-established pattern for library package-sets.